packages/shared/src/components/tooltips/ChangelogTooltip.tsx

```diff
@@ -99,138 +90,122 @@ function ChangelogTooltip<TRef extends HTMLElement>({
     await updateExtension();
   };
 
+  const onToggleUpvote = async () => {
+    await toggleUpvote({ post, origin: Origin.ChangelogPopup });
+  };
+
   return (
-    <BaseTooltip
-      {...props}
-      content={
-        !!post && (
-          <div
-            className="flex relative flex-col whitespace-normal break-words rounded-16 border shadow-2 focus:outline-none w-[24rem] changelog bg-theme-bg-tertiary border-theme-color-cabbage"
-            data-testid="changelog"
+    !!post && (
+      <InteractivePopup
+        {...props}
+        position={InteractivePopupPosition.Unset}
```

Comment:
More of a question, why did you choose unset and then manually add them?
My suggestion would be to add something like bottomLeft as a position potentially? (easier if we ever need to re-use the logic)

Reply:
Well, `BottomLeft` would be something similar to what is already there `LeftEnd` (bottom-8, left-8).  What we would want in this case is something set LeftEnd but only on the main feed container, the one already there takes the popup on top of the sidebar. The already specified bottom is also more then we need based on design (we need a bottom-6 rather then bottom-8)
We could create something like MainFeedLeftEnd (with `left-60` which is the sidebar size, and either bottom-6 or bottom-0 and mb-6), we would just not be picking from the already defined props there but would need to create new ones... my thoughts, at the moment this seems to be something quite specific to this use case, not reusable, thus the unset prop solution came to mind.

But maybe by not knowing the props that much I might be missing some other way how we might do this that makes sense and is more reusable, suggestions?

Reply:
I see LeftEnd is actually never used, probably was a proposal. so for all I care you can overwrite that one.
Just double check I didn't miss anything.

Reply:
LeftEnd is composed of `classNames(leftClass, endClass)`, both of those classes are used in other positions, so in this case that LeftEnd would need to be composed of these 2 new classes I need (left-60, bottom-6), which can make it confusing... If instead I do name it `MainFeedLeftEnd` I can create 2 classes, mainFeedLeft = left-60 and mainFeedEnd = bottom-6, which at least would make it a bit more clear.

Reply:
Sure let's do that just for re-usability :)